<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QWidget" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>320</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/pic/pic/GameBackGround.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_timeboard">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>離開</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Key_F">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Key_J">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>J</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_hittime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>194</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_scoreboard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QLabel" name="TimeTitle">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreTitle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="HitandMiss">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>164</y>
     <width>141</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Right/Wrong</string>
   </property>
  </widget>
  <widget class="QLabel" name="DrumLine">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>628</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/pic/pic/DrumLine.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="taikoAnimate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HitDrum">
   <property name="geometry">
    <rect>
     <x>-8</x>
     <y>87</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/pic/pic/hitdrum.png</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_wrongHitTime">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>194</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>290</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>再玩一次</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>background</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
